docs/how-to-provision-using-dps.md

```diff
@@ -216,19 +222,23 @@ namespace Samples.AdtIothub
 }
 ```
 
+Save the file and then re-publish your function app. For instructions on publishing the function app, see the [*Publish the app*](tutorial-end-to-end.md#publish-the-app) section of the end-to-end tutorial.
+
 ### Configure your function
 
-Next, you'll need to set environment variable in your function app from earlier, containing the reference to the Azure Digital Twins instance you've created. If you used the the end-to-end tutorial ([*Tutorial: Connect an end-to-end solution*](./tutorial-end-to-end.md)) your settings will already be configured, otherwise review both the setting up of the sample function app as well as assigning permissions to the function app.
+Next, you'll need to set environment variables in your function app from earlier, containing the reference to the Azure Digital Twins instance you've created.
+
+If you used the the end-to-end tutorial ([*Tutorial: Connect an end-to-end solution*](./tutorial-end-to-end.md)), your settings will already be configured.
+
+To set up the environment variable, you will need the Azure Digital Twins instance's **_host name_** from when you set up your instance. If you need to gather this value again, use [these instructions](how-to-set-up-instance-portal.md#verify-success-and-collect-important-values) from the setup article for finding them in the [Azure portal](https://portal.azure.com).
```

Comment:
I'd consider putting this in the prereqs section next to the instruction that they need an instance, and shifting the scope to "here are the values you'll need in the rest of this article from the step you just completed."

That will allow them to gather the info when they're still in the mindset of setting the instance up, before they move on to the mindset of completing the DPS connection steps, and you'll only have to write this section once instead of repeating it in both sections.

Reply:
Agree, we need to move these to the prerequisites and then reference them only in this section.